adapters/api/get-contract-asset.js

```diff
@@ -0,0 +1,93 @@
+import {
+  ATLAN_API_TOKEN,
+  ATLAN_INSTANCE_URL,
+} from "../utils/get-environment-variables.js";
+
+import fetch from "node-fetch";
+import {
+  getErrorAssetNotFound,
+} from "../templates/atlan.js";
+import stringify from "json-stringify-safe";
+
+export default async function getContractAsset({
+  dataset,
+  assetQualifiedName,
+}) {
+  var myHeaders = {
+    Authorization: `Bearer ${ATLAN_API_TOKEN}`,
+    "Content-Type": "application/json",
+  };
+
+  var raw = stringify(
```

Comment:
Should we also consider adding typeName to the filter to only look for assets that we support? 🤔

Reply:
Done